keyboards/dz60/config.h

```diff
@@ -44,6 +44,8 @@
 #    define RGBLIGHT_SAT_STEP 8
 #    define RGBLIGHT_VAL_STEP 8
 #    define RGBLIGHT_SLEEP
+#    define RGBLIGHT_LAYERS
+#    define RGBLIGHT_LAYERS_OVERRIDE_RGB_OFF
```

Comment:
This should be removed, and moved to a config.h file in your keymap folder, instead. 

```suggestion
```